Zend/zend_API.c

```diff
@@ -2769,8 +2769,8 @@ ZEND_API zend_result zend_register_functions(zend_class_entry *scope, const zend
 		}
 		lowercase_name = zend_string_tolower_ex(internal_function->function_name, type == MODULE_PERSISTENT);
 		lowercase_name = zend_new_interned_string(lowercase_name);
-		reg_function = malloc(sizeof(zend_internal_function));
-		memcpy(reg_function, &function, sizeof(zend_internal_function));
+		reg_function = calloc(1, sizeof(zend_function));
```

Comment:
That `calloc()` appears unnecessary, since we're overwriting the memory in the next statement anyway.